Month1/Week2/Section2/SpotifyProject/Playlist.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+* Instructions found in Playlist.h
+*/
+
+#include "Playlist.h"
+
+Playlist::Playlist()
+{
+	playlistTitle = "N/A";
+	songCount = 0;
+}
+
+Playlist::Playlist(std::string pPlaylistTitle)
+{
+	playlistTitle = pPlaylistTitle;
+	songCount = 0;
+}
+
+void Playlist::setPlaylistTitle(std::string pPlaylistTitle)
+{
+	playlistTitle = pPlaylistTitle;
+}
+
+std::string Playlist::getPlaylistTitle()
+{
+	return playlistTitle;
+}
+
+bool Playlist::addSong(Song song)
+{
+	if (song.isValid())
+	{
+		songs[songCount] = song;
+		songCount++;
+		return true;
+	}
+	else
```

Comment:
This else statement is not wrong but is redundant. Your if statement returns true and anything else should return false. So you could simply write
```
if (condition)
{
    // Do stuff
    return true;
}

return false;
```